modules/data-loading/pages/s3-loader-user-guide.adoc

```diff
@@ -150,70 +129,75 @@ DROP DATA_SOURCE *
 DROP DATA_SOURCE ALL
 ----
 
-==== SHOW DATA_SOURCE
+==== List data sources
 
-The SHOW DATA_SOURCE command will display a summary of all existing data_sources for which the user has privilege:
+The `SHOW DATA_SOURCE` command will display a summary of all existing data_sources for which the user has privilege:
 
 [,console]
 ----
 $ GSQL SHOW DATA_SOURCE *
 
-# The sample output:
+// The sample output:
 Data Source:
   - S3 s1 ("file.reader.settings.fs.s3a.access.key": "AKIAJ****4YGHQ", "file.reader.settings.fs.s3a.secret.key": "R8bli****p+dT4")
-# The global data source will be shown in global scope.
-# The graph scope will only show the data source it has access to.
+// The global data source will be shown in global scope.
+// The graph scope will only show the data source it has access to.
 ----
 
-=== 2. Create a Loading Job
+=== 2. Create a loading job
 
-The S3 Loader uses the same basic https://docs.tigergraph.com/dev/gsql-ref/ddl-and-loading/creating-a-loading-job#create-loading-job[CREATE LOADING JOB] syntax used for standard GSQL loading jobs. A DEFINE FILENAME statement should be used to assign a loader FILENAME variable to a S3 data source name and the path to its config file.
+The S3 Loader uses the same basic xref:gsql-ref:ddl-and-loading:creating-a-loading-job.adoc[CREATE LOADING JOB] syntax used for standard GSQL loading jobs.
```

Comment:
Do we want backticks to get monospacing even when CREATE LOADING JOB is a hyperlink? I could go either way but this is something we should be consistent on and I can't find another example one way or another.

Reply:
Yes, we should always put backticks over code. I have added them.